ras_party/models/models.py

```diff
@@ -213,7 +214,8 @@ def _get_business_associations(businesses):
         for business_respondent in businesses:
             business_dict = {
                 "partyId": business_respondent.business.party_uuid,
-                "sampleUnitRef": business_respondent.business.business_ref
+                "sampleUnitRef": business_respondent.business.business_ref,
+                "businessRespondentStatus": business_respondent.status.name
```

Comment:
Missing respondent:
"businessRespondentStatus": business_respondent.**respondent**.status.name

Reply:
Ooh that might be why it broke.